source/initial-dns-seedlist-discovery/tests/load-balanced/loadBalanced-directConnection-errors.yml

```diff
@@ -1,7 +0,0 @@
-# The TXT record for test20.test.build.10gen.cc contains loadBalanced=true. There is no corresponding test for
```

Comment:
Instead of deleting this test, can we instead change it so that it asserts that `directConnection=false` does not error?